src/NexusMods.App.Cli/OptionParsers/GameParser.cs

```diff
@@ -11,11 +11,33 @@ internal class GameParser(IGameRegistry gameRegistry) : IOptionParser<IGame>
 {
     public bool TryParse(string toParse, out IGame value, out string error)
     {
-        var install = gameRegistry.Installations.Values.FirstOrDefault(g => g.Game.GameId.ToString() == toParse) ??
-                    gameRegistry.Installations.Values.FirstOrDefault(g => g.Game.Name.Equals(toParse, StringComparison.CurrentCultureIgnoreCase))!;
-
-        value = (IGame)install.Game;
-        error = string.Empty;
-        return true;
+        try
+        {
+            var install = gameRegistry.Installations.Values.FirstOrDefault(g => g.Game.GameId.ToString() == toParse) ??
+                          gameRegistry.Installations.Values.FirstOrDefault(g => g.Game.Name.Equals(toParse, StringComparison.CurrentCultureIgnoreCase)) ??
+                          GameInstallation.Empty;
+            if (install.Equals(GameInstallation.Empty))
+            {
+                throw new NullReferenceException();
+            }
+            value = (IGame)install.Game;
+            error = string.Empty;
+            return true;
+        }
+        catch (NullReferenceException)
+        {
+            // Recheck if current game parse is supported
+            var install = gameRegistry.SupportedGames.FirstOrDefault(g => g.GameId.ToString() == toParse) ??
+                       gameRegistry.SupportedGames.FirstOrDefault(g => g.Name.Equals(toParse, StringComparison.CurrentCultureIgnoreCase))!;
+            if (install == null!)
+            {
+                value = null!;
+                error = $"Game '{toParse}' is not found or supported.";
+                return false;
+            }
+            value = (IGame)install;
+            error = $"Warning: Game '{toParse}' is supported but not installed. Please install it properly for full functionality.";
+            return true;
+        }
```

Comment:
Don't use exceptions for control flow, this can be simplified:

```suggestion
        var game = gameRegistry.SupportedGames.FirstOrDefault(g => g.Name.Equals(toParse, StringComparison.OrdinalIgnoreCase));
        if (game is null && uint.TryParse(toParse, out var parsedGameId))
        {
            game = gameRegistry.SupportedGames.FirstOrDefault(g => g.GameId.Equals(parsedGameId));
        }

        if (game is null)
        {
            value = null!;
            error = $"Unknown game: `{toParse}`";
            return false;
        }

        value = (IGame)game;
        error = string.Empty;
        return true;
```